src/Powercord/plugins/pc-translate/style.scss

```diff
@@ -2,8 +2,14 @@
   color: #00b0f4;
   cursor: pointer;
   margin-left: 3px;
+  font-size: 0.75rem;
 
   &:hover {
     text-decoration: underline;
   }
 }
+
+.pc-translate-reset-compact-container {
```

Comment:
We usually reserve `.pc-` css classes to normalized components, and custom components uses `.powercord-`

Reply:
changed